Dockerfile.layers

```diff
@@ -0,0 +1,4 @@
+FROM amazonlinux
```

Comment:
Occurred to me that this source container could be pinned and managed by dependabot... See: https://github.com/plus3it/ash-linux-formula/blob/master/tests/docker/centos7/Dockerfile

Reply:
updated